project/assets/scripts/component/component-constructor.js

```diff
@@ -1,24 +1,24 @@
 const ComponentConstructor = function (options) {
-    if (options.login) {
+    if (options === 'login') {
```

Comment:
In cases where you're doing repetitive checks against the same value, you can use `switch` / `case` constructs for cleaner code. Optional, for your consideration.

Reply:
I'll use switch, thanks!